custom-recipes/timeseries-forecast-1-train-evaluate/recipe.json

```diff
@@ -144,15 +145,17 @@
         {
             "name": "additional_columns",
             "label": "Long format",
-            "description": "If your dataset has columns to identify multiple time series",
+            "description": "If values from distinct time series are stored in
```

Comment:
I find the new version harder to understand than the previous one

Reply:
It's the 'official' description of long time format provided by Dataiku. As the user told us that is what not clear what it meant I wanted to try something else. But if you find it more unclear we can switch back, no problem for me

Reply:
To find a middle ground, how about "If multiple time series are stored in the same target column"?

Reply:
ok !

Reply:
🌞 
```suggestion
            "description": "If multiple time series are stored in the same column"
```